File: api/models/src/MerchantAccount.go
package models

import (
	"common"
)

// MerchantAccount Model说明
type MerchantAccount struct {
	ID            int     `json:"id" xorm:"id pk autoincr"`       // 编号
	MerchantID    int     `json:"merchant_id" xorm:"merchant_id"` //
	MerchantName  string  `json:"merchant_name"`
	AppCount      int     `json:"app_count"`
	Remain        float64 `json:"remain"`
	Frozen        float64 `json:"frozen"`
	Total         float64 `json:"total"`
	TotalIn       float64 `json:"total_in"`
	TotalOut      float64 `json:"total_out"`
	CountRemain   int     `json:"count_remain"`
	CountFrozen   int     `json:"count_frozen"`
	CountTotal    int     `json:"count_total"`
	SentTotal     int     `json:"sent_total"`
	SentSuccess   int     `json:"sent_success"`
	TotalRecharge float64 `json:"total_recharge"`
	TotalWithdraw float64 `json:"total_withdraw"`
	Updated       int64   `json:"updated"`

	*common.Model `xorm:"-"`
}

// MerchantAccounts 数据访问对象
var MerchantAccounts = MerchantAccount{
	Model: &common.Model{
		TabName: "merchant_accounts",
	},
}

// TableName 指定数据表名
func (ths *MerchantAccount) TableName() string {
	return "merchant_accounts"
}

File: php-api/app/Model/MerchantAccount.php
<?php

declare(strict_types=1);

namespace App\Model;

/**
 * @property int $id
 * @property int $merchant_id
 * @property string $merchant_name
 * @property float $remain
 * @property float $frozen
 * @property float $total
 * @property float $total_in
 * @property float $total_out
 * @property int $state
 * @property int $created
 * @property int $updated
 * @property int app_count
 * @property int order_count
 * @property int order_valid
 * @property int order_success
 * @property float order_total
 * @property float order_total_valid
 * @property float total_withdraw
 */
class MerchantAccount extends Model
{
    const CREATED_AT = 'created';
    const UPDATED_AT = 'updated';
    const STATUS_TYPES = [0 => '禁用', 1 => '正常'];
    protected ?string $table = 'merchant_accounts';
    protected ?string $dateFormat = 'U';
}
